ui.tests/test-module/specs/title/titleV2.authoring.spec.js

```diff
@@ -0,0 +1,138 @@
+            .should("exist");
+            cy.get('.cq-dialog-cancel').click();
+            cy.deleteComponentByPath(titleDrop);    
+        } else {
+            dropTitleInContainer();
+            cy.openEditableToolbar(sitesSelectors.overlays.overlay.component + titleEditPathSelector);
+            cy.invokeEditableAction("[data-action='CONFIGURE']");
+            cy.get("[name='./fd:htmlelementType']")
+            cy.get('[name="./fd:htmlelementType"]').eq(0).click()
+            .should("exist");
+            cy.get('.cq-dialog-cancel').click();
+            cy.get('[data-path^="/content/forms/af/core-components-it/blank/jcr:content/guideContainer/title"]')
+                .invoke('attr', 'data-path')
+                .then(dataPath => {
+                    const id = dataPath.replace('/content/forms/af/core-components-it/blank/jcr:content/guideContainer/title', '');
+                    const _titleDrop = titleDrop + id;
+                    cy.deleteComponentByPath(dataPath);
+                })
+        }
+
+    }
+
+    context('Open Forms Editor', function () {
+        const pagePath = "/content/forms/af/core-components-it/blank",
+            titleEditPath = pagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/title",
+            titleEditPathSelector = "[data-path^='" + titleEditPath + "']",
+            titleDrop = pagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/" + afConstants.components.forms.resourceType.title.split("/").pop();
+
+        beforeEach(function () {
+            // this is done since cypress session results in 403 sometimes
+            cy.openAuthoring(pagePath);
+        });
+
+        it('check edit dialog availability of Title', function () {   
+            cy.cleanTitleTest(titleEditPath).then(() => {
+                testTitleEditDialog(titleEditPathSelector, titleDrop, false);   
+            })
+        });
+        
+    });
+
+    context('Open Sites Editor', function () {
+        const pagePath = "/content/core-components-examples/library/adaptive-form/title",
+            titleEditPath = pagePath + afConstants.RESPONSIVE_GRID_DEMO_SUFFIX + "/guideContainer/title",
+            titleEditPathSelector = "[data-path='" + titleEditPath + "']",
+            editDialogConfigurationSelector = "[data-action='CONFIGURE']",
+            titleDrop = pagePath + afConstants.RESPONSIVE_GRID_DEMO_SUFFIX + '/guideContainer/' + afConstants.components.forms.resourceType.title.split("/").pop();
+
+        beforeEach(function () {
+            // this is done since cypress session results in 403 sometimes
+            cy.openAuthoring(pagePath);
+        });
+
+        it('insert aem forms Title', function () {
+            dropTitleInSites();
+            cy.deleteComponentByPath(titleDrop);
+        });
+
+        it('check edit dialog availability of Title', function () {
+            testTitleEditDialog(titleEditPathSelector, titleDrop, true);
```

Comment:
Can also write a runtime test case for v2 ?

Reply:
Done!